web/styles/compose.css

```diff
@@ -340,10 +340,26 @@
 .edit-content-container:has(.message_edit_content.over-limit:focus) {
     box-shadow: 0 0 0 1pt
         var(--color-message-content-container-border-over-limit);
+    background-color: var(
```

Comment:
Why is this needed when the background-color property is set below as well for `#compose-textarea`? For edit UI the background color should be set only to the input and not to the formatting area below like we do for the compose box.

Reply:
Now I understand what is the problem. I think I should write css separately for 

`#message-content-container:has(.new_message_textarea.over-limit)`,
`#message-content-container:has(.new_message_textarea.over-limit:focus)`,  and 

`.edit-content-container:has(.message_edit_content.over-limit)`,
`.edit-content-container:has(.message_edit_content.over-limit:focus)`.

and same applies for approaching-limit.

Reply:
Done!